docs/api-reference/next/image.md

```diff
@@ -255,26 +228,136 @@ Other properties on the `<Image />` component will be passed to the underlying
 
 - `style`. Use `className` instead.
 - `srcSet`. Use
-  [Device Sizes](/docs/basic-features/image-optimization.md#device-sizes)
+  [Device Sizes](#device-sizes)
   instead.
 - `ref`. Use [`onLoadingComplete`](#onloadingcomplete) instead.
 - `decoding`. It is always `"async"`.
 
-## Styling
+## Configuration Options
+
+### Set Default Loader
 
-`next/image` wraps the `img` element with a single `div` element to maintain the aspect ratio of the image and prevent [Cumulative Layout Shift](https://vercel.com/blog/core-web-vitals#cumulative-layout-shift).
+If you want to use a cloud provider to optimize images instead of using the Next.js built-in Image Optimization, you can configure the loader and path prefix. This allows you to use relative urls for the Image `src` and automatically generate the correct absolute url for your provider.
```

Comment:
```suggestion
If you want to use a cloud provider to optimize images instead of using the Next.js built-in Image Optimization API, you can configure the `loader` and `path` prefix. This allows you to use relative urls for the Image `src` and automatically generate the correct absolute url for your provider.
```